eclipse-theia/theia
# Fix web plugin initialization

#### What it does

Closes https://github.com/eclipse-theia/theia/issues/12786

Instead of registering a custom static route handler to `context`, we simply move the web plugin initialization file to `lib/frontend/context` so that it can be handled by the general static frontend file handler. This makes the file fetching independent of whether the application backend is currently bundled.

#### How to test

1. Start the application and ensure that `http://localhost:3000/context/plugin-vscode-init-fe.js` serves the file correctly.
2. Install a web plugin and assert that it works as expected. See [here](https://github.com/eclipse-theia/theia/pull/10721) for more info.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
